These thests require cloning https://github.com/AlDanial/cloc_submodule_test
skip them for now so we can run the tests without network access.

--- a/t/02_git.t
+++ b/t/02_git.t
@@ -89,14 +89,15 @@
 
 my $Verbose = 0;
 
+SKIP: {
 if (!-d 'cloc_submodule_test') {
     print "-" x 79, "\n";
     print "Directory 'cloc_submodule_test' is not found; git tests skipped.\n";
     print "To enable the tests, create the directory with\n";
     print "    git clone https://github.com/AlDanial/cloc_submodule_test.git\n";
-    ok( 0, "git tests");
+    skip("git tests");
     print "-" x 79, "\n";
-} else {
+}
     my $results  = 'results.yaml';
     my $work_dir = getcwd;
     my $cloc     = "$work_dir/../cloc";   # all-purpose version
